torchfix/torchfix.py

```diff
@@ -28,15 +29,16 @@
 
 ALL_VISITOR_CLS = [
     TorchDeprecatedSymbolsVisitor,
```

Comment:
Any ideas how to avoid this ugly duplication of all visitors?

Reply:
Yes, we can automatically pick them up with a registry pattern. Already got something in progress for this.